apps/pie-docs/src/components/chip/overview/overview.md

```diff
@@ -0,0 +1,326 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Chip
+    order: 1
+shouldShowContents: true
+---
+
+## Overview
+
+The purpose of chips is to provide a visual representation of a specific entity or attribute, such as a selected option or a labelled category. Chips can also be interactive, allowing users to remove or modify the selected choices.
+
+Chips are commonly used in various contexts, including filtering options, search results or any situation where concise and visually distinct information needs to be displayed.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/chip/overview.svg",
+    alt: "A group of chips that are horizontally stacked together."
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.list,
+        items: [
+            "Chips should appear dynamically as a group of multiple interactive elements."
+        ]
+    },
+    dont: {
+        type: usageTypes.list,
+        items: [
+            "Don’t use when an interaction is not required, use a [tag](/components/tag/) if needed."
```

Comment:
Just trying to make sure I understand the difference here between a tag (which is never interactive?) and a chip (which can possibly be non-interactive, if `isDismissible` is false)?

Would I be correct in thinking that a tag is more for describing an attribute of something, whereas a chip describes its state (with the implication that this can/does change)?

Reply:
Chip is always interactive, even if `isDismissible` is false.